broker/amqp/queue.go

```diff
@@ -39,8 +41,16 @@ func newQueue(pipe *jobs.Pipeline, lsn func(event int, ctx interface{})) (*queue
 		return nil, fmt.Errorf("missing `queue` parameter on amqp pipeline")
 	}
 
+	isValidExchangeType := map[string]bool{"direct": true, "fanout": true, "topic": true, "headers": true}
+	exchangeType := pipe.String("exchange-type", "direct")
+
+	if !isValidExchangeType[exchangeType] {
+		return nil, fmt.Errorf("unknown `exchange-type` parameter on amqp pipeline")
```

Comment:
Error message should not contain backquotes.
```suggestion
		return nil, fmt.Errorf("unknown exchange type parameter on amqp pipeline")
```